### PR TITLE
Monitoring: Fix runtime errors when monitoring Dataset Metadata

### PR DESCRIPTION
**Pull request by MrSnyder**
This fixes runtime errors when monitoring Dataset Metadata instances.

### CODE REVIEW

File: mrmap/monitoring/monitoring.py [hunk 1]
```diff
@@ -202,6 +203,17 @@ def check_service(self, url: str, check_wfs_member: bool = False, check_image: b
         else:
             self.handle_service_error(service_status)
```
Comment (jokiefer) [inline mrmap/monitoring/monitoring.py:205]: This should be better places as an utility function in resourceNew app.

**Review by jokiefer — CHANGES_REQUESTED** [review]
see comment

File: mrmap/monitoring/monitoring.py [hunk 2]
```diff
@@ -202,6 +203,17 @@ def check_service(self, url: str, check_wfs_member: bool = False, check_image: b
...
 
+    def _get_session_for_request(self):
```
Comment (jokiefer) [inline mrmap/monitoring/monitoring.py:206]: This should be better places as an utility function in resourceNew app.

**Review by jokiefer — CHANGES_REQUESTED** [review]
see comment

File: mrmap/locale/de/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -700,8 +700,8 @@ msgstr "Erlaubte Bereiche"
 #: acls/views.py:25 acls/views.py:28 registry/tables/metadata.py:16
 #: registry/tables/metadata.py:29 registry/tables/service.py:31
 #: registry/tables/service.py:83 registry/tables/service.py:129
-#: registry/tables/service.py:171 registry/views/monitoring.py:42
-#: registry/views/monitoring.py:64 users/views/groups.py:22
+#: registry/tables/service.py:171 registry/views/monitoring.py:46
+#: registry/views/monitoring.py:73 users/views/groups.py:22
 #: users/views/groups.py:47 users/views/groups.py:51
 msgid "Details"
 msgstr "Einzelheiten"
```
Comment (MrSnyder) [inline mrmap/locale/de/LC_MESSAGES/django.po:707]: "Einzelheiten" is a rather uncommon word in German, probably better: "Details"

**Review by MrSnyder — COMMENTED** [review]
LGTM, however after @jokiefer comment I am a bit unsure if `monitoring.py` is the best place for `_get_session_for_request`.

File: mrmap/locale/de/LC_MESSAGES/django.po (hunk 3 above)
Comment (Unraveler) [inline mrmap/locale/de/LC_MESSAGES/django.po:707]:
```suggestion
msgstr "Details"
```

**Pull request merged**